src/ex4/server.js

```diff
@@ -0,0 +1,57 @@
+// Express boilerplate, hosting the `dist` file, connecting to the routes
+const fs = require("fs");
+const express = require("express");
+
+require("express-async-errors");
+
+const livereload = require("livereload");
+const connectLiveReload = require("connect-livereload");
+
+const logger = require("./server/middleware/logger");
+
+const errorHandler = require("./server/middleware/error_handler");
+const itemRouter = require("./server/routes/itemRouter");
+const port = 8080;
+const server = express();
+
+const liveReloadServer = livereload.createServer();
+
+server.use(connectLiveReload());
+
+//
+server.use([logger, express.json(), express.static("dist")]);
+server.use("/item", itemRouter);
+
+server.get("/", (req, res) => {
+  res.status(200).json({
+    health: `ok`,
+  });
+});
+
+server.post("/error", async (req, res, next) => {
```

Comment:
what's the purpose of this?
should not have such things in code - an external API function to throw error?
Also it throw error in code and then catches it and invoke the `next(e)`?